ios/FluentUI/Drawer/DrawerController.swift

```diff
@@ -606,7 +624,8 @@ open class DrawerController: UIViewController {
     private func updatePreferredContentSize(isExpanded: Bool) {
         isPreferredContentSizeBeingChangedInternally = true
         if isExpanded {
-            preferredContentSize.height = UIScreen.main.bounds.height
+            let expandedContentHeight = (preferredMaximumExpansionHeight < UIScreen.main.bounds.height && preferredMaximumExpansionHeight >= originalDrawerHeight) ? preferredMaximumExpansionHeight : UIScreen.main.bounds.height
```

Comment:
it doesn't seem like you are using "expandedContentHeight" variable. how did you create it?

Reply:
please check the current changes, please let me know if it looks good now.